eb_sqs_ecs_project/README.md

```diff
@@ -0,0 +1,60 @@
+# AWS Service 1 to AWS Service 2
```

Comment:
Please fix!

Reply:
Thank you. I've made the changes. Please review and let me know if any more updates are required.